src/api/goods.ts

```diff
@@ -10,10 +10,16 @@ export function getAll(): Promise<Good[]> {
 
 export const get5First = () => {
   return getAll()
-    .then(goods => goods); // sort and get the first 5
+    .then(goods => {
```

Comment:
yes, it was written with `then`, but you are an engineer, let's rewrite it to async & await 🙂